src/FixedDataTable.js

```diff
@@ -957,8 +957,13 @@ var FixedDataTable = createReactClass({
       children.push(child);
     });
 
+    var tableWidth = props.width;
+    var scrollContentHeight = this._scrollHelper.getContentHeight();
+
     // Allow room for the scrollbar, less 1px for the last column's border
-    var adjustedWidth = props.width - Scrollbar.SIZE - Scrollbar.OFFSET;
+    if (props.showScrollbarY && props.height < scrollContentHeight) {
```

Comment:
Is this check correct?  Isn't there also a need to take into account the reserved height for the header and such.

e.g. props.height < scrollContentHeight + reservedHeight